node/client/src/lib.rs

```diff
@@ -373,6 +374,35 @@ impl sc_client_api::StorageProvider<Block, crate::FullBackend> for Client {
 			}
 		}
 	}
+
+	fn closest_merkle_value(
```

Comment:
what are these?

Reply:
pls add 1-2 lines what these 2 new methods do

Reply:
Hi, Substrate's `sc_client_api` `v1.2` interface requires these to be implemented.